src/components/TimeseriesChart.js

```diff
@@ -110,8 +109,7 @@ class TimeseriesChartComponent extends Component {
     );
 
     this.setState({
-      combinedEvents,
-      wantedAxes: axes
```

Comment:
Hi Remco,
Valt op dat je behalve updaten van Plotly ook de wantedAxis uit de state van het Timeseries Component haalde.
Dit lijkt me goed, maar weet je misschien nog waarom deze uberhaupt in de state stond? Was dit misschien een workaround voor een probleem?
Denk dat het code technisch zeker een verbetering is om dit uit de state te halen, maar weet niet hoe we in dit geval zeker weten dat we niet een nieuwe bug introduceren. We kunnen natuurlijk door de git history van Parramatta heen gaan.
Wat denk jij?